src/gui/editors/PianoRoll.cpp

```diff
@@ -3499,11 +3499,15 @@ void PianoRoll::paintEvent(QPaintEvent * pe )
 			// is the note in visible area?
 			if (note->key() > bottomKey && note->key() <= topKey)
 			{
-				// we've done and checked all, let's draw the note
+				// We've done and checked all, let's draw the note with
+				// the appropriate color
+				QColor fillColor = note->type() == Note::RegularNote ? m_noteColor : m_stepNoteColor;
```

Comment:
Use ``const auto``. Once again, mostly subjective, but knowing the type here is not too important (we actually do know it already by intuition, lol).
```suggestion
				const auto fillColor = note->type() == Note::RegularNote ? m_noteColor : m_stepNoteColor;
```